src/types/YMap.js

```diff
@@ -74,7 +99,7 @@ export class YMap extends AbstractType {
    */
   _integrate (y, item) {
     super._integrate(y, item)
-    ;/** @type {Map<string, any>} */ (this._prelimContent).forEach((value, key) => {
+    this._prelimContent?.forEach((value, key) => {
```

Comment:
note this logic change, but if `_prelimContent` was null previously then this would have created a runtime error.